HandsOn/Group04/mappings/mappings-with-links-updated.yml

```diff
@@ -0,0 +1,102 @@
+prefixes:
+    rr: http://www.w3.org/ns/r2rml#
+    foaf: http://xmlns.com/foaf/0.1/
+    xsd: http://www.w3.org/2001/XMLSchema#
+    rdfs: http://www.w3.org/2000/01/rdf-schema#
+    dc: http://purl.org/dc/elements/1.1/
+    rev: http://purl.org/stuff/rev#
+    gtfs: http://vocab.gtfs.org/terms#
+    geo: http://www.w3.org/2003/01/geo/wgs84_pos#
+    owl: http://www.w3.org/2002/07/owl#
+    schema: http://schema.org/
+    dct: http://purl.org/dc/terms/
+    rml: http://semweb.mmlab.be/ns/rml#
+    ql: http://semweb.mmlab.be/ns/ql#
+    pproc: http://contsem.unizar.es/def/sector-publico/pproc#
+    rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+    dp: http://dbpedia.org/ontology/
+    organ: http://www.w3.org/ns/org#
+    pp: http://exercise.publicprocurement.es/about/ontology/group04pp/
+    
+mappings:
+  Department:
+    sources:
+      - [purchase-order-2014-2015-updated-with-links.csv~csv]
+    s: dp:Department#$(Department Name)
+    po:
+      - [a, dp:Department]
+      - [foaf:name, $(Department Name),xsd:string]
+      - [owl:sameAs, $(wikidataDepartment)~iri]
+  ContractObject:
+    sources:
+      - [purchase-order-2014-2015-updated-with-links.csv~csv]
+    s: pproc:$(Normalized UNSPSC)
```

Comment:
The use of : here is wrong, IMO

Reply:
It is also unclear why you use UNSPSC codes for the resources.

Reply:
Regarding the use of UNSPSC, we discussed it before because if you add the name directly, its URI will look very strange. Because the name directly has a lot of spaces, like this.
<http://contsem.unizar.es/def/sector-publico/pproc#PAPER%2CCOPY%2C20%20LB%2C8-1%2F2%22%20X%2011%22%2CWHT> <http://xmlns.com/foaf/> 0.1/name> "PAPER,COPY,20 LB,8-1/2\" X 11\",WHT"^^<http://www.w3.org/2001/XMLSchema#string>.

Translated with www.DeepL.com/Translator (free version)

Reply:
This should be the same as the previous error and should be changed to.
pproc:/$(Normalized UNSPSC)

Regarding the use of UNSPSC, we discussed it before because if you add the name directly, its URI will look very strange or ugly. Because the name directly has a lot of spaces, like this.
“<http://contsem.unizar.es/def/sector-publico/pproc#PAPER%2CCOPY%2C20%20LB%2C8-1%2F2%22%20X%2011%22%2CWHT> <http://xmlns.com/foaf/> 0.1/name> "PAPER,COPY,20 LB,8-1/2\" X 11\",WHT"^^<http://www.w3.org/2001/XMLSchema#string>.”

Reply:
http://yourowndomain/resource/unspsc/$(Normalized UNSPSC)

Reply:
It would be better to use the code, and not the text associated to the code. For example:
http://yourowndomain/resource/unspsc/2344423423